sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedSource.cs

```diff
@@ -68,6 +68,7 @@ internal DataFeedDetailPatch InstantiateDataFeedDetailPatch()
         {
             return Parameter switch
             {
+                /*
                 AzureApplicationInsightsParameter p => new AzureApplicationInsightsDataFeedPatch() { DataSourceParameter = p },
```

Comment:
Same here. Service split an existing model into two different types, so I conversion will be needed.